common/src/main/java/tachyon/exception/ExceptionMessage.java

```diff
@@ -116,6 +116,9 @@
   LINEAGE_INPUT_FILE_NOT_EXIST("The lineage input file {0} does not exist"),
   LINEAGE_OUTPUT_FILE_NOT_EXIST("No lineage has output file {0}"),
 
+  // client
+  INCOMPATIBLE_VERSION("{0}Client version {1} is not compatible with server version {2}"),
```

Comment:
The {0} argument is passed in using getServiceName(), that uses a constant defined in `Constants` that does not have the Client suffix.